fe/src/components/Header/Header.tsx

```diff
@@ -43,6 +47,15 @@ export default function Header(): JSX.Element {
             ))}
           </HStack>
         </HStack>
+        <HStack spacing={12} ml="auto">
+          <IconButton
```

Comment:
Could you add on-hover here that will say either `Switch to light theme` or `Switch to dark theme`